stable/prometheus-mongodb-exporter/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: "v0.7.0"
+appVersion: "v0.7.1"
```

Comment:
Don't bump the `appVersion`, bump the `version`

Reply:
Opps ! Fixed